great_expectations/expectations/registry.py

```diff
@@ -268,7 +269,7 @@ def get_metric_kwargs(
     metric_name: str,
     configuration: Optional[ExpectationConfiguration] = None,
     runtime_configuration: Optional[dict] = None,
-) -> Dict:
+) -> dict:
```

Comment:
Is this as specific as we can get? At a glance I would think this returns something like `Dict[str, Union[str, MetricValue]]`.

Reply:
@NathanFarmer I am not sure; but for now, I just wanted to remove the abuse of type that does not indicate anything meaningful.  I would propose that we handle this comprehensively via a JIRA item.  I have been wanting to correct this for a long time.  Thanks.